lib/phoenix_live_view/router.ex

```diff
@@ -365,6 +366,69 @@ defmodule Phoenix.LiveView.Router do
     end
   end
 
+  @doc """
+  Applies Liveview session data.
+
+  This plug must be placed **after** the `:protect_from_forgery` plug.
+  """
+  def fetch_liveview_session(conn, _opts \\ []) do
```

Comment:
I would come up with a generic name that does both the session and the flash, so we still need only one helper in the router.